Documentation/getting-started.md

```diff
@@ -0,0 +1,112 @@
+
+`Places` is the mobile SDK supporting the Location Service.
+
+## Configure the Places extension in Data Collection UI
+
+1. In the Data Collection UI, from your mobile property, select the **Extensions** tab.
+2. On the **Catalog** tab, locate or search for the **Places** extension, and select **Install**.
+3. Select the **POI Library (or libraries)** you wish to use in the app.
+4. Select **Save**.
+5. Follow the publishing process to update SDK configuration.
+
+![Places extension configuration](./assets/config.png)
+
+## Add the AEP Places extension to your app
+
+## Download and import the Places extension
+
+1. Add the Mobile Core and Places extensions to your project using the app's Gradle file.
+
+```java
+implementation 'com.adobe.marketing.mobile:core:2.+'
+implementation 'com.adobe.marketing.mobile:places:2.+'
+```
+
+> **Warning**
+> Using dynamic dependency versions is not recommended for production apps. Refer to this [page](https://github.com/adobe/aepsdk-core-android/blob/main/Documentation/GradleDependencies.md) for managing gradle dependencies.
+
+2. Import the Mobile Core and Places extensions in your application's main activity.
+
+#### Java
+
+```java
+import com.adobe.marketing.mobile.MobileCore;
+import com.adobe.marketing.mobile.Places;
+```
+
+#### Kotlin
+
+```kotlin
+import com.adobe.marketing.mobile.MobileCore
+import com.adobe.marketing.mobile.Places
+```
+
+## Register the Places extension with Mobile Core
+
+Call the `setApplication()` method once in the `onCreate()` method of your main activity.
+
+For example, your code might look like the following:
+
+#### Java
+
+```java
+public class MainApp extends Application {
+    private static final String APP_ID = "YOUR_APP_ID";
+
+    @Override
+    public void onCreate() {
+        super.onCreate();
+
+        MobileCore.setApplication(this);
+        MobileCore.configureWithAppID(APP_ID);
+
+        List<Class<? extends Extension>> extensions = new ArrayList<>();
+        extensions.add(Places.EXTENSION);
+        MobileCore.registerExtensions(extensions, o -> {
+            Log.d(LOG_TAG, "AEP Mobile SDK is initialized");
+        });
+    }
+
+}
+```
+
+#### Kotlin
+
+```kotlin
+class MyApp : Application() {
+
+    override fun onCreate() {
+        super.onCreate()
+        MobileCore.setApplication(this)
+        MobileCore.configureWithAppID("YOUR_APP_ID")
+
+        val extensions = listOf(Places.EXTENSION)
+        MobileCore.registerExtensions(extensions) {
+            Log.d(LOG_TAG, "AEP Mobile SDK is initialized")
+        }
+    }
+
+}
+```
+
+## Implement Places APIs
+
+For more information about implementing Places extension APIs, please read the [Places API reference](./api-usage.md).
+
+
+## Configuration keys
+
+To update SDK configuration programmatically, use the following information to change your Places configuration values. For more information, see the [Configuration API reference](https://github.com/adobe/aepsdk-core-android).
```

Comment:
provide the configurationAPI link ? 
 https://github.com/adobe/aepsdk-core-android/blob/main/Documentation/MobileCore/api-reference.md#updating-the-configuration-programmatically

Reply:
i think the idea is to not create broken links in the future.  so rather than pointing directly to a specific api on a page, we just point to the repo.